controllers/watcher_common.go

```diff
@@ -36,6 +36,9 @@ var (
 const (
 	// TransportURLSelector is the name of key in the secret created by TransportURL
 	TransportURLSelector = "transport_url"
+	// ServicePasswordSelector is the name of key in the internal Secret for
+	// the watcher service password
+	ServicePasswordSelector = "ServicePassword"
```

Comment:
I don't see this constant used anywhere. The key for the watcher service password in the secret is WatcherPassword (and I'd leave it this way and avoid using this new constant, tbh).